cluster/src/multi-jvm/scala/akka/cluster/ClusterSpec.scala

```diff
@@ -0,0 +1,162 @@
+package akka.cluster
+
+import java.util.concurrent.ConcurrentHashMap
+
+import akka.actor.{ActorIdentity, ActorSystem, Address}
+import akka.remote.testconductor.Controller.ClientDisconnected
+import akka.remote.testconductor.RoleName
+import akka.remote.testkit.MultiNodeSpec
+import akka.testkit.EventFilter
+import akka.testkit.TestEvent.Mute
+import kanaloa.reactive.dispatcher.STMultiNodeSpec
+import scala.concurrent.duration._
+
+//adapted from https://github.com/akka/akka/blob/master/akka-cluster/src/multi-jvm/scala/akka/cluster/MultiNodeClusterSpec.scala
+trait ClusterSpec extends  STMultiNodeSpec { self: MultiNodeSpec =>
```

Comment:
Not sure I follow your question. Did you mean that why not having `ClusterAwareBackendSpec` extending this `ClusterSpec`?